src/card.tsx

```diff
@@ -465,7 +466,7 @@ const CardContent: React.FC<CardContentProps> = ({
 type TouchableProps = {
     trackingEvent?: TrackingEvent | ReadonlyArray<TrackingEvent>;
 } & ExclusifyUnion<
-    | {href: string | undefined; newTab?: boolean}
+    | {href: string | undefined; newTab?: boolean; onPress?: undefined}
```

Comment:
why is this `onPress?: undefined` needed? `ExclusifyUnion` should do the trick here

Reply:
leftovers, I was just trying some things